spec/libraries/aws_engine_spec.rb

```diff
@@ -93,57 +93,57 @@ def contents
 
   describe 'resource_endpoint' do
```

Comment:
__[reek]__
* [UnusedParameters] has unused parameter 'params' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)